### PR TITLE
css-layout-classes - Michael Suh

### PR DESCRIPTION
**Pull request by michael-suh**
css layout classes SR

### CODE REVIEW

File: css-layout-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+.row {
+  display: flex;
+  width: 100%;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+  padding-left: 8px;
+  padding-right: 8px;
+  justify-content: center;
```
Comment (codingcodymiller) [inline css-layout-classes/styles.css:16]: You can delete these.

File: css-layout-classes/styles.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.column-one-fourth {
+  flex-basis: calc(100% / 4);
+  padding-left: 8px;
+  padding-right: 8px;
+  justify-content: center;
```
Comment (codingcodymiller) [inline css-layout-classes/styles.css:23]: You can delete these.

File: css-layout-classes/styles.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.column-two-thirds {
+  flex-basis: calc(100% / 1.5);
+  padding-left: 8px;
+  padding-right: 8px;
+  justify-content: center;
```
Comment (codingcodymiller) [inline css-layout-classes/styles.css:30]: You can delete these.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by michael-suh** [comment]
css layout classes SR

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**